### DISCUSSION TITLE
My Windows11 Bonjour is unable to resolve embassy.local to my embassies 192.168.0.10 address.

### DISCUSSION THREAD

**Discussion by djangofan** [Start9Labs/start-os#2144] [Q&A] [answered]
My Windows11 Bonjour is unable to resolve embassy.local to my embassies 192.168.0.10 address. 

Accessing https://192.168.0.10 works fine and I can get into my Embassy but some things will not work for me without the name resolution that matches the trusted certificate!

I tried a lot of things to fix, in this order, but I cannot get it to work.    Granted, my experience with Windows11 is new and evidently what I know would work on Windows7, no longer works for me on Windows11.     Maybe this thread can be a guide for Windows users?

1.   I tried completely uninstalling my virus software, thinking that it was causing a blockage.
2.   I tried adding this entry to my  system32/drivers/etc/hosts file:
           192.168.0.10  embassy.local
          
       This had no effect on the name resolution.  I even ran the ifconfig command to refresh dns cache.  I also tried restart of system.
3.   I installed Bonjour service and it appears to be running fine.  Also, I have latest iTunes installed.
4.    My Embassy is configured with the network name of "embassy"
5.   My computer has no issues auto-detecting printers on my network
6.    My  hard wired network adapter connects directly to my router.  My DNS server is 8.8.8.8  with a alternate DNS server of 192.168.0.1

That's all I can think of.  Any ideas?

**Comment by djangofan** [comment]
I just realized I did edit the settings and changed the name of my Embassy from the DEFAULT name that it had.   Maybe that has no affect on how the machine is located?   Maybe I need to use the original name the Embassy got at setup time?  I forgot the name unfortunately.  How can I get it?      I suppose I can try to SSH to the box if i can find that documentation.

I guess I can completely re-install the Node, but i will have to resync Bitcoin.

Asked question here too: https://superuser.com/questions/1766486/my-windows11-machine-is-unable-to-resolve-mdns-on-my-network

**Comment by kn0wmad** [comment]
If you are able to access your device by IP address, then you can go to the Embassy tab -> About in order to get your unique id (.local) and Tor (.onion) addresses.  The unique id is not changed when editing the 'name' of the device.  You will need to access your embassy-unique-id.local address - access via IP is not currently supported or required, and should only be used for troubleshooting.

**Comment by djangofan** [comment, marked as answer]
Ok, that fixed it.   I was assuming that renaming the Embassy would also change my .local DNS name.   This was not obvious.
Got it working.

By the way, if this helps anyone else ever, here is my mDNS config on my Windows11 machine:

PS C:\Users\jausten> Get-NetUDPEndpoint -LocalPort 5353 | Select-Object LocalAddress,LocalPort,OwningProcess,@{ Name="ProcessName"; Expression={((Get-Process -Id $_.OwningProcess).Name )} }

LocalAddress  LocalPort OwningProcess ProcessName
------------  --------- ------------- -----------
::1                5353          5480 mDNSResponder
::                 5353         13736 msedge
192.168.159.1      5353          5760 nvcontainer
192.168.137.1      5353          5760 nvcontainer
192.168.0.5        5353          5760 nvcontainer
172.27.144.1       5353          5480 mDNSResponder
172.22.128.1       5353          5760 nvcontainer
0.0.0.0            5353          2908 svchost

**Reply by kn0wmad** [reply]
Glad you're up and running, thank you for reaching out

**Comment by djangofan** [comment]
Addendum:
On Window11, after importing the cert as "User root cert" (rather than System root cert),  I had to restart Chrome browser completely before it recognizes the cert.
